Community-Supported/clouddb-extractor/azuresql_extractor.py

```diff
@@ -151,24 +152,56 @@ def hyper_sql_type(self, source_column: Any) -> SqlType:
             return_sql_type = SqlType.numeric(source_column_precision, source_column_scale)
         else:
             return_sql_type = type_lookup.get(source_column_type)
-
             if return_sql_type is None:
                 error_message = "No Hyper SqlType defined for MySQL source type: {}".format(source_column_type)
                 logger.error(error_message)
                 raise HyperSQLTypeMappingError(error_message)
-
             return_sql_type = return_sql_type()
 
         logger.debug("Translated source column type {} to Hyper SqlType {}".format(source_column_type, return_sql_type))
         return return_sql_type
 
+    def _column_type_override(self, column_name)-> Optional[SqlType]:
+        """
+        For some SQL Server implementations precision,scale may be reported incorrectly 
+        and this can be overridden by specifying override_column_type in config.yml
+        
+        Sample Usage:
+            override_column_type:
+                "column_a":
+                    sqltype: "numeric"
+                    precision: 18
+                    scale: 3
+                "column_b" : 
+                    sqltype: "int"
+        """
+        col_type=None
+        override_column_type=self.source_database_config.get("override_column_type")
+        if override_column_type:
+           this_override=override_column_type.get(column_name)
+           if this_override:
+            try:
```

Comment:
indent is weird here (a single space?)
Would appreciate if you could run pyformat (or some other auto-formatting tool) over this complete file